source/language/delays.rst

```diff
@@ -38,31 +38,31 @@ It is often useful to reference the duration of other parts of the
 circuit. For example, we may want to delay a gate for twice the duration
 of a particular sub-circuit, without knowing the exact value to which
 that duration will resolve. Alternatively, we may want to calibrate a
-gate using some pulses, and use its duration as a new ``length`` in order to delay
-other parts of the circuit. The ``lengthof()`` intrinsic function can be used for this
+gate using some pulses, and use its duration as a new ``duration`` in order to delay
+other parts of the circuit. The ``durationof()`` intrinsic function can be used for this
 type of referential timing.
 
-Below are some examples of values of type ``length``.
+Below are some examples of values of type ``duration``.
 
 .. code-block:: c
 
-       // fixed length, in standard units
-       length a = 300ns;
-       // fixed length, backend dependent
-       length b = 800dt;
-       // fixed length, referencing the duration of a calibrated gate
-       length c = lengthof(defcal);
-       // dynamic length, referencing a box within its context
-       length d = lengthof(box);
-
-We further introduce a ``stretch`` type which is a sub-type of ``length``. Stretchable lengths
-have variable non-negative length that is permitted to grow as necessary
+       // fixed duration, in standard units
+       duration a = 300ns;
+       // fixed duration, backend dependent
+       duration b = 800dt;
+       // fixed duration, referencing the duration of a calibrated gate
+       duration c = durationof(defcal);
```

Comment:
I would replace this with `durationof({x $3})` and then drop the next one.

Reply:
Addressed in https://github.com/Qiskit/openqasm/pull/204/commits/40df87657528bc9c5aa65355cc79175c01763fc7.